tests/abi/tools/revng-ensure-rft-equivalence/Main.cpp

```diff
@@ -216,7 +216,7 @@ int main(int Argc, char *Argv[]) {
   LeftModel.replaceReferences(LeftReplacements);
   RightModel.replaceReferences(RightReplacements);
 
-  // Remove all the dynamic functions so that they don't interfere
+  // Remove all the dynamic functions so that they don't interfere.
```

Comment:
Ehr, we try to go the other way around. Trailing `.` in single-line comments should be banned.

Reply:
(I just realized I forgot to close a couple of threads on Friday)

This one is done.